### PR TITLE
Upgraded compiling documentation with description of ELMAH 2x

### PR DESCRIPTION
**Pull request by ThomasArdal**

### CODE REVIEW

**Comment by atifaziz** [comment]
@ThomasArdal Thanks, better against a branch! :+1:

Do you want to try & squash 95bb2591498ee3dac540deed7b958236c13ada8c into 022cf1447aa2b88a3dccb97829504ec68053d31b? If you're not sure, I can do it on my end but then I'll do close this PR & do it offline.

File: Compiling.md [hunk 1]
```diff
@@ -5,23 +5,42 @@ permalink: a/compiling/
 article: true
 ---
 
+A lot changed from ELMAH 1x to ELMAH 2x, why this document is split into two parts.
+
+## ELMAH 2x
```
Comment (atifaziz) [inline Compiling.md]: This is the most current info so doesn't need to be under a heading.

File: Compiling.md [hunk 2]
```diff
@@ -5,23 +5,42 @@ permalink: a/compiling/
...
+
+You can compile ELMAH from the sources in one of two ways:
```
Comment (atifaziz) [inline Compiling.md]: I think we need to simplify this whole area. It made sense to talk about building from the command line & VS previously when VS solutions were not MSBuild files.

Comment (ThomasArdal) [inline Compiling.md]: Well, there's two ways to build ELMAH, right? From Visual Studio and MSBuild (through the cmd file). Don't you want to explain the two options here?

File: Compiling.md (hunk 1 above)
Comment (ThomasArdal) [inline Compiling.md]: Fixed

**Comment by ThomasArdal** [comment]
@atifaziz I'm terrible at rebasing :smile: Just tried to squash, but I get an error message saying "coult not apply 95bb259". If you're an expert, it would be great if you could squash the commits that you don't want.

**Comment by atifaziz** [comment]
Merged in and closed by 5d49c5d192b8f98dfc4cc9e639734b4c7ece3b9a

**Comment by atifaziz** [comment]
[Published](https://elmah.github.io/a/compiling/).

**Comment by ThomasArdal** [comment]
Awesome. Feel free to make the changes that you mentioned :smile:

Den ons. 16. mar. 2016 21.49 skrev Atif Aziz <notifications@github.com>:

**Pull request closed without merging**